client/scheduler_test.go

```diff
@@ -30,8 +27,20 @@ func TestCreateDeleteAndFindScheduler(t *testing.T) {
 	expectedScheduler.StartDate = scheduler.StartDate
 	expectedScheduler.StartTime = scheduler.StartTime
 
-	if !reflect.DeepEqual(*scheduler, expectedScheduler) {
-		t.Errorf("The scheduler does not match what we expected. actual: %v expected: %v", *scheduler, expectedScheduler)
+	if !reflect.DeepEqual(scheduler, expectedScheduler) {
+		t.Errorf("The scheduler does not match what we expected. actual: %v expected: %v", scheduler, expectedScheduler)
+	}
+	fmt.Println("")
+	fmt.Println(expectedScheduler)
+	fmt.Println(scheduler)
+	fmt.Println("")
```

Comment:
This looks like debugging code that is no longer needed

Reply:
Removed.